main.js

```diff
@@ -0,0 +1,43 @@
+const userData = []
+
+const showUsers = function () {
+
+    $.ajax({
+        url: 'https://randomuser.me/api/?results=10',
+        dataType: 'json',
+        success: function (data) {
+            // userData.first = data.results[0].name.first;
+            // userData.last = data.results[0].name.last;
+            // userData.email = data.results[0].email;
+            // userData.id = data.results[0].id.value || 666;
+        
+            for (user of data.results) {
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"